code.js

```diff
@@ -1,26 +1,47 @@
-window.onload = function () { // Run this once the page has loaded.
-    // search
-    document.querySelector("#searchButton").addEventListener("click", searchGithub);
-    function searchGithub() {
-        const searchUserText = document.querySelector("#searchUser").value;
-
-        // Obtain a list of users from the Github API that match searchUserText
-        //  The final result will contain an array under the key 'items'
-        // Pass this array to `renderUserList`
+window.onload = function () {
+  // Run this once the page has loaded.
+  // search
+  document
+    .querySelector("#searchButton")
+    .addEventListener("click", searchGithub);
+  function searchGithub() {
+    const searchUserText = document.querySelector("#searchUser").value;
+    if (searchUserText == "") {
```

Comment:
make sure not to put an empty `if` block. if you are only doing something in the `else` block, just use negative logic in the `if` and remove the `else` block entirely. So this would be
`if (searchUserTest !== ""){`
`the code currently in your else block`
`}`

Reply:
Oh crap that makes so much more sense. LOL